pkg/controller/localvolumeset/add.go

```diff
@@ -80,5 +82,38 @@ func AddLocalVolumeSetReconciler(mgr manager.Manager) error {
 		return err
 	}
 
+	// Watch for changes to secondary resource PersistentVolume and enqueue the LocalVolumeSet
```

Comment:
Again, please add a comment *why* do you enqueue other objects that use the same storage class.

Reply:
Added